app/controllers/orders/items.js

```diff
@@ -22,6 +22,19 @@ export default Ember.Controller.extend(SearchMixin, {
     };
   },
 
+  scannedItem: Ember.observer("searchInput", function() {
```

Comment:
I'd recommend naming `observers` something like `scannedItemWatcher` as opposed to `scannedItem`

Otherwise it can very quickly become confusing, or conflict with actual properties of the controller : )

Reply:
Yes will do that @patrixr